pkg/filter/filter_engine_test.go

```diff
@@ -28,47 +30,60 @@ func TestFilterEngine_Run(t *testing.T) {
 			name: "filter on type",
 			expr: "Type=='filtered_resource'",
 			resources: []resource.Resource{
-				&testresource.FakeResource{},
 				&testresource.FakeResource{
-					Type: "filtered_resource",
+					CtyVal: &cty.DynamicVal,
```

Comment:
You can remove all theses dynamic val

Reply:
I didn't realise that with `bytes, _ := ctyjson.Marshal(*ctyVal, ctyVal.Type())` it crashes if I don't put at least a cty.Value inside the FakeResource